File: src/main/java/personal/bs/service/SellerService.java
package personal.bs.service;


import personal.bs.domain.po.StorePO;
import personal.bs.domain.vo.PageResult;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface SellerService {

    /**
     * 返回全部列表
     *
     * @return
     */
    public List<StorePO> findAll();


    /**
     * 返回分页列表
     *
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize);


    /**
     * 增加
     */
    public void add(StorePO seller);


    /**
     * 修改
     */
    public void update(StorePO seller);


    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    public StorePO findOne(Integer id);


    /**
     * 批量删除
     *
     * @param ids
     */
    public void delete(Integer[] ids);

    /**
     * 分页
     *
     * @param pageNum  当前页 码
     * @param pageSize 每页记录数
     * @return
     */
    public PageResult findPage(StorePO seller, int pageNum, int pageSize);


    public void updateStatus(String sellerId, String status);
}

File: src/main/resources/static/js/service/sellerService.js
//服务层
app.service('sellerService',function($http){

    this.updateStatus=function(id,status){
        return $http.get('../store/updateStatus?id='+id +'&status='+status);
    }
	this.login=function(entity){
        return  $http.post('../store/login',entity );
    }
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../store/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../store/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../store/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../store/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../store/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../store/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../store/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});
